Dockerfile

```diff
@@ -8,8 +10,7 @@ COPY conjur-cli.gemspec conjur-cli.gemspec
 COPY lib/conjur/version.rb lib/conjur/version.rb
 
 # Make sure only one version of bundler is available
-RUN gem uninstall bundler -i /usr/local/lib/ruby/gems/2.1.0 bundler || true && \
-  gem uninstall bundler -i /usr/local/lib/ruby/gems/2.2.0 bundler || true && \
+RUN gem uninstall bundler -i /usr/local/lib/ruby/gems/${RUBY_VERSION}.0 bundler || true && \
   gem uninstall bundler -aIx && \
   gem install bundler -v 1.11.2 && \
```

Comment:
Will be great to get rid of this old bundler version dependency. 

I don't remember the issue that made us pin it, do you? A future issue/story, perhaps.